spec/css-spec.coffee

```diff
@@ -84,16 +84,41 @@ describe 'CSS grammar', ->
         expect(tokens[0]).toEqual value: '.- ', scopes: ['source.css', 'meta.selector.css']
 
     describe 'id selectors', ->
-      it 'tokenizes a standalone id selector', ->
-        {tokens} = grammar.tokenizeLine '#ch-1 {}'
+      it 'tokenizes \\#unicorn as id selector', ->
```

Comment:
Surprise: I'm 99% sure that this test will still be skipped.  No matter how much escaping you do, Atom will still "helpfully" skip the test for you.